include/layers/InputLayer.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <algorithm>
+#include <cmath>
+#include <cstdlib>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <stdexcept>
+#include <string>
+
+#include "layers/Layer.hpp"
+
+class InputLayer : public Layer {
+ public:
+  InputLayer() = default;
+  static void run(const std::vector<std::string>& path, Tensor& output) {
+    std::vector<int> res(path.size() * 227 * 227 * 3);
+    for (int num = 0; num < static_cast<int>(path.size()); num++) {
+      cv::Mat image = cv::imread(path[num]);
+      if (image.empty()) {
+        throw std::runtime_error("Failed to load image");
+      }
+      cv::Mat resized_image;
+      cv::resize(image, resized_image, cv::Size(227, 227));
+      std::vector<cv::Mat> channels;
+      cv::split(resized_image, channels);
```

Comment:
This also should be outside of input layer. Image should be read and converted to Tensor to be processed by input layer